job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/publisher/PostBuildScriptsContext.groovy

```diff
@@ -6,10 +6,13 @@ import javaposse.jobdsl.dsl.DslContext
 import javaposse.jobdsl.dsl.Item
 import javaposse.jobdsl.dsl.JobManagement
 import javaposse.jobdsl.dsl.helpers.step.StepContext
+import javaposse.jobdsl.dsl.RequiresPlugin
```

Comment:
The warning says `no jobManagement field in javaposse.jobdsl.dsl.helpers.publisher.PostBuildScriptsContext`. The `@RequiresPlugin` annotation needs access to `JobManagement` to determine the installed plugins. Change `PostBuildScriptsContext` to inherit from `AbstractContext` and you get the `jobManagement` field.